pages/profile/views/index.jsx

```diff
@@ -29,6 +29,7 @@ class Index extends React.Component {
 
     const model = this.props.model;
     const hasEstablishments = !!model.establishments.length;
+    const formatDate = (date, format) => (date ? dateFormatter(date, format) : '-');
```

Comment:
You should define this function outside of the component so it is defined once globally instead of on every render.

Reply:
Done.